applications/CompressiblePotentialFlowApplication/tests/test_CompressiblePotentialFlowApplication.py

```diff
@@ -31,6 +33,8 @@ def AssembleTestSuites():
     # smallSuite will contain the following tests:
     # - testSmallExample
     smallSuite = suites['small']
+    smallSuite.addTest(Naca0012SmallTest('test_execution'))
+    smallSuite.addTest(SmallLiftJumpTest('test_execution'))
```

Comment:
And this would be replaced by: 

```suggestion
    smallSuite.addTest(PotentialFlowTests('test_SmallLiftJumpTest'))
```

Reply:
Thank you for your time and your comments! I will try to resolve them!